extensions/amp-analytics/0.1/instrumentation.js

```diff
@@ -88,7 +88,7 @@ class AnalyticsEvent {
 
   /**
    * @param {!AnalyticsEventType|string} type The type of event.
-   * @param {!Object<string, string>=} opt_vars A map of vars and their values.
```

Comment:
Confused. How does ! and = work together? How is it different from ? and = being used together?